framework/build.mk

```diff
@@ -57,6 +57,7 @@ ifneq (,$(findstring mpi,$(cxx_compiler)))
 	cxx_compiler = $(shell $(libmesh_CXX) -show)
 endif
 
+
```

Comment:
Thanks @dschwen. There are also a few unnecessary changes to Moose.C in these commits, probably due to merge conflict resolutions, which we'll get cleaned up before merging.